ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/ExternalBackupIntegrations/S3IntegrationForm.tsx

```diff
@@ -331,6 +334,7 @@ function S3IntegrationForm({
                     </FormLabelGroup>
                     <FormLabelGroup
                         label=""
+                        labelIcon={s3IamRole()}
```

Comment:
This one is not rendering for me. 🤔

Reply:
hmm let me try to set an explicit label in the form group. If that doesn't work we'll need to get the cavalry and ask the UI experts 😅

Reply:
seems to have done the trick.